QUESTIONS.md

```diff
@@ -1,13 +1,6 @@
 # Questions about Code, Node, Express, React
 
-
-What allows for the front end to connect with the back end ?
-
-What is a wiggle sort and when would it matter?
-
-What is the significance of "id=root" ?
```

Comment:
`root` is the name of the `id` inside of the index.html file and we pass the code into it from app.js

This is what the index.html looks like
`<div id="root"><div>` Note that there is nothing in it because it comes from app.js